pkgs/tools/security/dkimpy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, pythonPackages, fetchurl, openssl, makeWrapper }:
+
+with pythonPackages; buildPythonApplication rec {
+  name = "${pname}-${majorversion}.${minorversion}";
+  pname = "dkimpy";
+  majorversion = "0.6";
+  minorversion = "1";
+
+  src = fetchurl {
+    url = "https://launchpad.net/${pname}/${majorversion}/${majorversion}.${minorversion}/+download/${name}.tar.gz";
+    sha256 = "0zmvyw18ww1jqrbqws636w3xki59fyqva553r6s74q5c4jvy36v2";
+  };
+
+  buildInputs = with pythonPackages; [ pytest ];
+
+  patchPhase = ''substituteInPlace dknewkey.py --replace \
+     /usr/bin/openssl ${openssl}/bin/openssl
+     '';
+
+  propagatedBuildInputs =  [ openssl dns ];
+
+  postInstall = ''
+    mkdir -p $out/bin $out/libexec
+    mv $out/bin/*.py $out/libexec
+    makeWrapper "$out/libexec/dkimverify.py" $out/bin/dkimverify
+    makeWrapper "$out/libexec/dkimsign.py" $out/bin/dkimsign
+    makeWrapper "$out/libexec/arcverify.py" $out/bin/arcverify
+    makeWrapper "$out/libexec/arcsign.py" $out/bin/arcsign
+    makeWrapper "$out/libexec/dknewkey.py" $out/bin/dknewkey
```

Comment:
The packages for other distributions don't seem to have this renaming stuff (neither the included `.spec` in upstream nor the one one ArchLinux AUR). I think it would be better to not derivate from upstream here unless absolutely necessary.